torch/fx/experimental/proxy_tensor.py

```diff
@@ -580,6 +563,9 @@ def __init__(self, module: torch.fx.GraphModule, new_graph: torch.fx.Graph, deco
         super().__init__(module, **kwargs)
         self.new_graph = new_graph
         self.tracer = torch.fx.proxy.GraphAppendingTracer(self.new_graph)
+        # Blegh
```

Comment:
Why?

Reply:
I'm manually bonging in some attributes here, which should have been handled by the class itself